R/rcpp.R

```diff
@@ -25,6 +26,41 @@ use_rcpp <- function(name = NULL) {
   invisible()
 }
 
+#' @rdname use_rcpp
+#' @export
+use_rcpp_armadillo <- function(name = NULL) {
+  use_rcpp(name)
+
+  use_dependency("RcppArmadillo", "LinkingTo")
+
+  makevars_path <- proj_path("src/Makevars")
+  makevars_win_path <- proj_path("src/Makevars.win")
+
+  makevars_settings <- c(
+    "CXX_STD = CXX11",
+    "PKG_CXXFLAGS = $(SHLIB_OPENMP_CXXFLAGS)",
+    "PKG_LIBS = $(SHLIB_OPENMP_CXXFLAGS) $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)"
+  )
+
+  create_makevars(makevars_path, "RcppArmadillo", makevars_settings)
```

Comment:
I think `create_makevars()` should figure out the paths to the makevars, and should probably just write both files.